### PR TITLE
Improve `aleph.http/start-server`'s docstring for `:idle-timeout`

### LINKED ISSUE #609: Clarification regarding aleph.http/start-server's :idle-timeout option
The docstring for `aleph.http/start-server`'s `:idle-timeout` option states:

> when set, forces keep-alive connections to be closed after an idle time, in milliseconds

I think this is too narrow: AFAICT the timeout also applies to non-keep-alive connections. Can you confirm this? If so, I would clarify the docstring accordingly.

### PR DESCRIPTION
**Pull request by DerGuteMoritz**
Explain what exactly idle means and remove needlessly narrow and
potentially misleading reference to keep-alive connections.

Closes #609.

### CODE REVIEW

**Review by arnaudgeiser — APPROVED** [review]

**Comment by arnaudgeiser** [comment]
Removing the fact it was related to `keep-alive connections` is important here.

**Comment by KingMob** [comment]
@arnaudgeiser You have merging permissions, right?

**Comment by arnaudgeiser** [comment]
Yes, I have the merging permissions but I was also waiting for your approval there.

**Pull request merged**
